test/integration/functional-test.js

```diff
@@ -13,32 +13,261 @@ chai.use(chaiAsPromised);
 
 const ident = "lockbox_mozilla_com";
 
-describe("Lockbox functional testing", () => {
-  let webext, helper;
+// Note: Update these selectors if the DOM changes in components
+const makeSelectors = ({ By }) => ({
```

Comment:
Kind of figured these would change as UI work progresses. Could have followed the page object pattern, but forgot about it until just a little while ago. I can rework this if needed.